allscr/manual_tweaks/allscr.mrg_0139.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/allscr/retimed/allscr.mrg_0139.txt b/allscr/retimed/allscr.mrg_0139.txt
+--- allscr.mrg_0139.txt
++++ allscr.mrg_0139.txt
+@@ -559,7 +559,6 @@ _WKST(C847,eq,0);
+ _WKST(C859,eq,0);
+ _WNTY(0);
+ _ZM31615($010129);
+-_WKAD(C847,1);
```

Comment:
One of the scenes in this file is egregiously long, and has some forced space between the lines. This removes the forced space so that I can just _barely_ squeeze all the text in.